Components/Badge/badge.css

```diff
@@ -78,3 +78,26 @@
     min-width: 1.1rem;
     height: 1.1rem;
 }
+.text-badge{
+    color: var(--badge-text-color);
+    background-color: var(--badge-background-color);
+    padding: 0.3rem 0.6rem 0.3rem 0.6rem;
```

Comment:
For less than 1rem, you should be using px values.

Reply:
But according to my resarch this would work right..